tests/manual_e2e/basic_checks.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Helper function - prints an error message and exits
+exit_error() {
+    echo "Error: $*"
+    exit 1
+}
+
+echo "Checking if loki app is in deployed state"
+
+deployed=$(kubectl get app -n giantswarm loki -o yaml | yq .status.release.status)
+
+[[ "$deployed" != "deployed" ]] && exit_error "loki app is not in deployed state. Please fix the app before retrying" || echo "loki app is indeed in deployed state"
```

Comment:
The `||` is useless: if the test matches, it will exit_error.
Also, most linters don't like too long lines.

So I suggest something like this:
```suggestion
[[ "$deployed" != "deployed" ]] \
    && exit_error "loki app is not in deployed state. Please fix the app before retrying"
echo "✔️ loki app is in deployed state"
```
Which would apply to all tests then.

Reply:
Sure. As for the `||` this is just to keep logging updates on the advancement of the process. But if you think this isn't needed, I can get rid of those as well

Reply:
I didn't say the log message informing that the test passed successfully is useless.
I said the condition before it is useless.
My suggestion keeps the message, but removes the condition.

Reply:
Oh ok I get it

Reply:
@hervenicol I added back the messages without the condition